sdks/python/setup.py

```diff
@@ -30,9 +28,15 @@
 from pkg_resources import DistributionNotFound
 from pkg_resources import get_distribution
 from pkg_resources import normalize_path
+from pkg_resources import parse_version
 from pkg_resources import to_filename
 from setuptools import Command
 
+# distutils from stdlib is deprecated from python 3.10.
+# setuptools has been maintaining a copy of distutils and importing
+# setuptools will replace stdlib/distutils with setuptools/distutils.
```

Comment:
Add: 
# It is recommended to import setuptools prior to importing distutils to avoid
# using legacy behavior from distutils.
# https://setuptools.readthedocs.io/en/latest/history.html#v48-0-0

Reply:
Done